tools/bug-report/pkg/bugreport/bugreport.go

```diff
@@ -331,7 +331,9 @@ func BuildClientsFromConfig(kubeConfig []byte) (kube.Client, error) {
 
 func logAndPrintf(format string, a ...interface{}) {
 	fmt.Printf(format, a...)
-	log.Infof(format, a...)
+	o := []interface{}{format}
+	o = append(o, a...)
+	log.Infof(format, o)
```

Comment:
This is not safe, say if one writes

`logAndPrintf("hello i am %s,  version %s", "istio", "1.8.0")`